MediaBrowser.Model/MediaBrowser.Model.csproj

```diff
@@ -58,6 +58,7 @@
     <PackageReference Include="SmartAnalyzers.MultithreadingAnalyzer" PrivateAssets="All" />
   </ItemGroup>
   <ItemGroup>
+    <FrameworkReference Include="Microsoft.AspNetCore.App" />
```

Comment:
Is this necessary?

Reply:
I can check again but without it I was unable to compile. Afaik it's required to use IPNetwork

Reply:
This is required because I moved `IPData` to `MediaBrowser.Model` to work around an otherwise circular dependency. `IPData` requires `IPNetwork` from `Microsoft.AspNetCore.HttpOverrides`.

Reply:
Why can't you just pull it in as a dep?

Reply:
You mean as a `PackageReference`? I haven't done that ebcaus eit raises a warning:
```
warning NETSDK1080: A PackageReference to Microsoft.AspNetCore.App is not necessary when targeting .NET Core 3.0 or higher. If Microsoft.NET.Sdk.Web is used, the shared framework will be referenced automatically. Otherwise, the PackageReference should be replaced with a FrameworkReference.
```

Reply:
No, HttpOverrides